src/pages/uxp-api/reference-js/Global Members/ImageBlob/index.md

```diff
@@ -1,2 +1,189 @@
-# ImageBlob
-* [ImageBlob](ImageBlob.md)
+
```

Comment:
@kareemmyk  Move the content of ImageBlob.md to index.md in torq-native

Reply:
This is a limitation of the source code for docs generation. Once we group a file into a folder, then a master ```index.md``` get's created by default
So at this stage the current state is our best option